### PR TITLE
New dashboard sorted by TG and able to be edited in Excel

### PR DESCRIPTION
**Pull request by DBees**

### CODE REVIEW

File: program/dashboard/Dashboard_SpreadSheetFriendly.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## Status Key																												
+**PC = Project Concept** 																												
+&nbsp; &#8595;  																												
+**PL = Project Launch**  																												
+&nbsp; &#8595;  																												
+**PA = Plan Approved**																												
+&nbsp; &#8595;  																												
+**PF = Project Freeze**																												
+																												
+				<br /> +
```
Comment (MikeOpenHWGroup) [inline program/dashboard/Dashboard_SpreadSheetFriendly.md:14]: This doesn't look right.  I _think_ you are trying to add a line break here.  If that is the case it needs to start at the beginning of a line.

Comment (DBees) [reply program/dashboard/Dashboard_SpreadSheetFriendly.md]: @MikeOpenHWGroup thanks, fixed. Also I think @rickoco is busy so please merge this PR if you approve. (Lots more work needs done but we need to get the new structure out there. )

File: program/dashboard/Dashboard_SpreadSheetFriendly.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Status Key																												
+**PC = Project Concept** 																												
+&nbsp; &#8595;  																												
+**PL = Project Launch**  																												
+&nbsp; &#8595;  																												
+**PA = Plan Approved**																												
+&nbsp; &#8595;  																												
+**PF = Project Freeze**																												
+																												
+				 +
```
Comment (MikeOpenHWGroup) [inline program/dashboard/Dashboard_SpreadSheetFriendly.md:14]: The "+" character is extraneous.  The Markdown rendering on GitHub doesn't know how to handle it.  Recommend deleting this line altogether.

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]

**Review by MikeOpenHWGroup — APPROVED** [review]
@DBees, please issue another PR to eliminate that plus sign on line 14.

**Pull request merged**